### ISSUE TITLE
Cannot find module '../src/services'

### ISSUE THREAD

**Issue by travis-r6s** [Julien-R44/adonis-sail#4]
After installing this package, and running `node ace configure adonis-sail`, I get this errror thrown:

```sh
Cannot find module '../src/services'
Require stack:
- /home/travis/projects/<project>/node_modules/.pnpm/adonis-sail@0.0.5/node_modules/adonis-sail/build/commands/InstallSail.js
```

Which makes sense - in `/node_modules/adonis-sail/build/commands/InstallSail.js`, there is a line that requires the file `const services_1 = require("../src/services");`, but this file is not included in the NPM package:

![image](https://user-images.githubusercontent.com/12587127/172258148-5f0701dd-d110-4d3a-8c37-be1e128935f3.png)
![image](https://user-images.githubusercontent.com/12587127/172258216-ee68172a-2c7b-47a7-8e69-48c5ce4e53f5.png)

**Comment by travis-r6s** [comment]
If it is just a case of adding the services folder to the package JSON files array, I can create a PR to do this...

**Comment by Julien-R44** [comment]
Oops! Thanks for reporting. Fixed in 0.0.6

**Comment by travis-r6s** [comment]
Works great now - thanks! 🙌
